pkg/apis/kops/validation/legacy.go

```diff
@@ -268,6 +271,30 @@ func ValidateCluster(c *kops.Cluster, strict bool) *field.Error {
 		}
 	}
 
+	// Check CoreDNS.ServerIP
+	if c.Spec.CoreDNS != nil {
+		serverIPString := c.Spec.CoreDNS.ServerIP
+		if serverIPString == "" {
+			return field.Required(fieldSpec.Child("CoreDNS", "ServerIP"), "Cluster did not have CoreDNS.ServerIP set")
```

Comment:
FYI you don't have to include the field path in the message, because `fieldSpec.Child("CoreDNS", "ServerIP")` means it should be presented nicely.  _should_ be ;-)